.github/workflows/bump-release-version.yml

```diff
@@ -0,0 +1,41 @@
+#############################################################################
+# GitHub Action to bump release version
+#
+#############################################################################
+name: "Bump version"
+on:
+  workflow_dispatch:
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  bump:
+    name: Bump version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Modify File
+        run: |
+          ./.ci/bump-release-version.sh
```

Comment:
I do not see that you take new version from tag name.
This action should be triggered by tag.

Reply:
Changes Done